docs/developer/devicesimulator.md

```diff
@@ -0,0 +1,55 @@
+# Device Simulator
+
+`Devicesimulator` is used to simulate the load generated by devices.
+
+### Setup
+
+Copy the necessary setup files to run a `devicesimulator`.
+
+    mkdir -p ~/.flightctl/certs/ ~/.config/flightctl/
+    cp bin/agent/etc/flightctl/certs/* ~/.flightctl/certs/
+    cp bin/agent/etc/flightctl/config.yaml ~/.flightctl/agent.yaml
+
+### Running
+
+    bin/devicesimulator --count=1
+
+# Remote simulator
+
+To simulate a huge number of devices (10,000 for example) a
+`devicesimulator` has to be installed on a number of separate hosts to let it use more resources. Each remote host must be [set up](#remote-setup) with the same files. Then you have to [start simulator](#remote-run) on each host.
+
+### Remote setup
+
+Run the same commands per each host. Note: some commands must be run remotely.
+
+    # on remote host
+    mkdir -p ~/.flightctl/certs/ ~/.config/flightctl ~/bin
+    scp bin/agent/etc/flightctl/certs/* your_username@remotehoist:~/.flightctl/certs/
+    scp bin/agent/etc/flightctl/config.yaml your_username@remotehoist:~/.flightctl/agent.yaml
+    scp ~/.config/flightctl/client.yaml your_username@remotehoist:~/.config/flightctl/client.yaml
+    scp bin/devicesimulator your_username@remotehoist:~/bin/
+    # on remote host 
+    chmod -R 755 ~postgres/.flightctl/ ~postgres/.config/flightctl bin/devicesimulator
```

Comment:
See above.

As follow-up work, we should probably run `devicesimulator` from a K8s Deployment instead, so we can simply scale this out and have K8s take care of placement onto different hosts.